doc/nrf/gs_modifying.rst

```diff
@@ -181,41 +281,34 @@ Configuring build types
 
 .. build_types_overview_start
 
-Build types enable you to use different sets of configuration options for each board.
-You can create several build type :file:`.conf` files per board and select one of them when building the application.
-This means that you do not have to use one :file:`prj.conf` file for your project and modify it each time to fit your needs.
+When the ``CONF_FILE`` variable contains a single file and this file follows the naming pattern :file:`prj_<buildtype>.conf`, then the build type will be inferred to be *<buildtype>*.
+The build type cannot be set explicitly.
+The *<buildtype>* can be any string, but it is common to use ``release`` and ``debug``.
 
-.. build_types_overview_end
+For information about how to set variables, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
 
-.. note::
-    Creating build types and selecting them is optional.
-    This is a feature specific to the :ref:`application development in nRF Connect SDK <app_build_system>`.
+In case there are other configuration files that you would like to modify, such as Partition Manager :ref:`static configuration <ug_pm_static>` or any child image configuration, you need to create a separate configuration file for them using the same naming pattern.
+For example, the file for the Partition Manager static configuration would be called :file:`pm_static_release.yaml`.
```

Comment:
```suggestion
The static Partition Manager configuration can also be made dependent on the build type. When the build type has been inferred the file :file:`pm_static_<buildtype>.yml` will have precedence over :file:`pm_static.yml`.
```

We need to be explicit if the build type changes the static yaml file used, or if it just changes the order in which static yaml files are checked for existence.